modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -1041,6 +1042,13 @@ def get_oneof_instance(self, model_args, constant_args):
     # Iterate over each oneOf schema and determine if the input data
     # matches the oneOf schemas.
     for oneof_class in self._composed_schemas['oneOf']:
+        if oneof_class is none_type:
+            # The only way the input data can match the 'None' type
+            # is if the input data is the 'null' value.
+            if model_args is None:
+                # The input data matches the 'None' type.
+                oneof_instances.append(model_args)
```

Comment:
Can you delete this `if model_args is None:` block?
We will still go through this code path for deserializing pojo payloads but we should continue through the None like you are doing.
- this also applies to get_anyof_instances

Reply:
done